### PR TITLE
fix: add variable assignment checking as part of MinVersion

### PR DESCRIPTION
**Pull request by nanikjava**
Fix for #528

This PR update the assignment checking for `MinVersion` of TLS

### CODE REVIEW

File: rules/tls.go [hunk 1]
```diff
@@ -85,7 +86,27 @@ func (t *insecureConfigTLS) processTLSConfVal(n *ast.KeyValueExpr, c *gosec.Cont
 			}
 
 		case "MinVersion":
-			if ival, ierr := gosec.GetInt(n.Value); ierr == nil {
+			if ident, ok := n.Value.(*ast.Ident); ok {
+				if valueSpec, ok := ident.Obj.Decl.(*ast.ValueSpec); ok {
+					if ival, ok := valueSpec.Values[0].(*ast.SelectorExpr); ok {
+						x := ival.X.(*ast.Ident).Name
+						sel := ival.Sel.Name
+						fmt.Println(x, sel)
```
Comment (ccojocar) [inline rules/tls.go:94]: The `Println` should be removed before merging.

File: rules/tls.go [hunk 2]
```diff
@@ -85,7 +86,27 @@ func (t *insecureConfigTLS) processTLSConfVal(n *ast.KeyValueExpr, c *gosec.Cont
...
+
+						for _, imp := range c.Pkg.Imports() {
+							if imp.Name() == x {
+								typeObject := imp.Scope().Lookup(sel)
+								cnst, _ := typeObject.(*types.Const)
+								fmt.Print(cnst.Val().String())
```
Comment (ccojocar) [inline rules/tls.go:100]: ditto

**Review by ccojocar — CHANGES_REQUESTED** [review]
Please could you also try to fix the lint warning. I think not all all related to your changes. Thanks

**Comment by nanikjava** [comment]
@ccojocar Done all the changes you request and added test cases.

**Review by ccojocar — APPROVED** [review]
Thanks @nanikjava for fixing my comments!

**Pull request merged**